src/main/java/Exceptions/ChatIDMissingFromRequestURLParameterException.java

```diff
@@ -0,0 +1,20 @@
+package Exceptions;
+
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+
+public class ChatIDMissingFromRequestURLParameterException extends RuntimeException {
+
+    public static HttpStatus httpStatus = HttpStatus.BAD_REQUEST;
+    public static String message = "Parameter: 'chatID' Missing From Request URL";
+    public String path;
```

Comment:
For these too.
Please apply for all such classes.

Reply:
Done